src/services/ui/src/components/Layout/index.tsx

```diff
@@ -11,35 +11,45 @@ import { UsaBanner } from "../UsaBanner";
 import { useUserContext } from "../Context";
 import * as DropdownMenu from "@radix-ui/react-dropdown-menu";
 import config from "@/config";
-import { useNavigate, FAQ_TAB } from "../Routing";
+import { useNavigate } from "../Routing";
 import { ModalProvider, AlertProvider } from "@/components";
+import { isFaqPage, isProd } from "@/utils";
 
-const getLinks = (isAuthenticated: boolean, role?: boolean) => {
-  const isProd =
-    (window && window.location.hostname === "mako.cms.gov") ||
-    window.location.hostname === "onemac.cms.gov";
-  return [
-    {
-      name: "Home",
-      link: "/",
-      condition: true,
-    },
-    {
-      name: "Dashboard",
-      link: "/dashboard",
-      condition: isAuthenticated && role,
-    },
-    {
-      name: "FAQ",
-      link: "/faq",
-      condition: true,
-    },
-    {
-      name: "Webforms",
-      link: "/webforms",
-      condition: isAuthenticated && !isProd,
-    },
-  ].filter((l) => l.condition);
+const useGetLinks = () => {
```

Comment:
For future revisions to anything auth/user based, we could mimic how we built the rules system for Actions -- namely building an array of objects with a value and condition, then filtering where conditions fail. For this it'd look like an array of all possible nav links, and then a set of conditions mapped to them that measure a user's auth, the path, etc. as needed.